doc/documentation_data_format.rst

```diff
@@ -494,23 +494,33 @@ Detailed field description
 
   Scale of the parameter to be used during parameter estimation.
 
+  ``lin``
+    Use the parameter value, ``lowerBound``, ``upperBound``, and
+    ``nominalValue`` without transformation.
+  ``log``
+    Take the natural logarithm of the parameter value, ``lowerBound``,
+    ``upperBound``, and ``nominalValue`` during parameter estimation.
+  ``log10``
+    Take the logarithm to base 10 of the parameter value, ``lowerBound``,
+    ``upperBound``, and ``nominalValue`` during parameter estimation.
+
 - ``lowerBound`` [NUMERIC]
 
   Lower bound of the parameter used for estimation.
   Optional, if ``estimate==0``.
-  Must be provided in linear space, independent of ``parameterScale``.
+  The provided value will be transformed according to ``parameterScale``.
```

Comment:
```suggestion
  The provided value should be untransformed, as it will be transformed according to ``parameterScale`` during parameter estimation.
```

I guess this is trying to provide the following information:
- the simulator sees untransformed values
- the optimizer sees transformed values
- untransformed parameter values should be specified by the user here